assets/prototype/domain/eventEditor/datesList/dateCard/DeleteDateButton.tsx

```diff
@@ -1,9 +1,12 @@
 import * as React from 'react';
 import { Button } from '@blueprintjs/core';
-import useDeleteDatetimeMutation from '../../containers/mutations/useDeleteDateMutation';
 
-const DeleteDateButton = ({ eventId, id }) => {
-	const deleteDatetime = useDeleteDatetimeMutation({ eventId, id });
+// import useDeleteDatetimeMutation from '../../containers/mutations/useDeleteDateMutation';
+import useEntityMutator from '../../containers/mutations/useEntityMutator';
+
+const DeleteDateButton = ({ id }) => {
+	// const deleteDatetime = useDeleteDatetimeMutation({ eventId, id });
+	const { deleteEntity } = useEntityMutator('Datetime', id);
```

Comment:
let's delete the old commented out code since I'm assuming that will not be used anymore

Reply:
I left that just in case some changes may be needed. Will remove that.